lenskit-api/src/main/java/org/lenskit/api/ItemBasedItemRecommender.java

```diff
@@ -29,62 +26,68 @@
 
 
 /**
- * The interface for recommendation based on the items only. The difference from {@link
- * ItemRecommender} is that the input is only the item or list of items instead of user specific
- * information. This interface can be used to provide a Find Similar Items / People Also Liked
- * feature.
+ * Recommends items that go with a set of reference items. This interface is distinguished from
+ * {@link ItemRecommender} in that it uses a set of reference items instead
+ * of a user as the basis for computing scores. This component can be used to provide a Find Similar Items
+ * or People Also Liked feature.
  *
  * @see ItemRecommender
  * @author <a href="http://www.grouplens.org">GroupLens Research</a>
  * @compat Public
  * @since 0.10
  */
-public interface GlobalItemRecommender {
+public interface ItemBasedItemRecommender {
     /**
      * Recommend all possible items for a basket of items using the default exclude set.
      *
-     * @param items The items value.
+     * @param basket The reference basket.
      * @return The recommended items.
-     * @see #globalRecommend(Set, int, Set, Set)
+     * @see #recommendRelatedItems(Set, int, Set, Set)
      */
-    List<ScoredId> globalRecommend(Set<Long> items);
+    List<Long> recommendRelatedItems(Set<Long> basket);
 
     /**
      * Recommend up to <var>n</var> items for a basket of items using the default exclude set.
```

Comment:
Hmm. Yes, we should decide (and apply the same decision to `ItemRecommender`).

I like the idea that saying 'recommend(user)` just Does The Right Thing in the simple case. Code can replace `ItemRecommender` to redefine what 'right thing' is (might even introduce a new component to compute default candidate and exclude sets, so that it's easier for applications to redefine Do The Right Thing - a `CandidateIdentifier`).